src/main/resources/META-INF/plugin.xml

```diff
@@ -1,7 +1,12 @@
 <idea-plugin require-restart="true">
+    <resource-bundle>CodyBundle</resource-bundle>
```

Comment:
This entry was missing in my first attempt (https://github.com/sourcegraph/jetbrains/pull/988).